docs/application/web/api/6.5/ui_fw_api/Wearable_UIComponents/wearable_checkbox.htm

```diff
@@ -16,14 +16,6 @@ <h1>Checkbox and Radio Button</h1>
 <h2>Checkbox</h1>
 <img src="../../images/checkbox-wearable.png" alt="Photo of Checkbox in Tizen">
 <br>
-   <button
-      onclick="
-         if (navigator.userAgent.toLowerCase().indexOf('chrome') == -1) {
-            alert('Chrome Browser required')
-         } else
-            window.open('https://code.tizen.org/demos?path=1.1%2Fexamples%2Fwearable%2FUIComponents%2Fcontents%2Fcontrols%2Fcheckbox.html', '_blank')"
-   >Use example</button>
-   </button>
 <br>
 <h2>Radio Button</h2>
 <br>
```

Comment:
I found the button in line 32~38 is not removed. 
Would you please check it with the author of the PR? If the button should be removed also, please ask the author to raise the fix PR.
If the fix PR is merged to master, this PR 1671 will automatically include the fix PR.

Same issue to 6.0.

If the author says it is OK to include the button, I'll approve PR 1671.

Reply:
Thanks for your checking.
I also remove the button and make PR for it by https://github.com/Samsung/tizen-docs/pull/1672

Reply:
@mijin85cho the PR [1672 ](https://github.com/Samsung/tizen-docs/pull/1672) has been merged.